pkg/virt-api/webhooks/mutating-webhook/mutators/migration-create-mutator.go

```diff
@@ -51,6 +51,13 @@ func (mutator *MigrationCreateMutator) Mutate(ar *v1beta1.AdmissionReview) *v1be
 		return webhookutils.ToAdmissionResponseError(err)
 	}
 
+	// Add a special label
+	if migration.Labels == nil {
+		migration.Labels = map[string]string{"special": migration.Spec.VMIName}
+	} else {
+		migration.Labels["special"] = migration.Spec.VMIName
```

Comment:
we should give this label a more kubevirt specific name so we don't accidently collide with someone else's custom label

maybe, `kubevirt.io/vmi-name`. We define labels like this in `staging/src/kubevirt.io/client-go/api/v1/types.go`

Reply:
Sounds good, I'll change the label to `kubevirt.io/vmi-name`.
My inspiration for "special" was the one we set on VMIs, like in: https://github.com/kubevirt/kubevirt/blob/2fbf85bf88adec3e4ca8ee907d6c2bcc72004739/examples/vmi-ephemeral.yaml#L6

For upgrades, setting the label in the virt-controllers migration reconcile loop sounds like a good plan, I'll work on that today.